src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1311,9 +1311,13 @@ public function searchProductsAction(Request $request): JsonResponse
             $currencyIsoCode = $currencyId !== null
                 ? Currency::getIsoCodeById((int) $currencyId)
                 : Currency::getIsoCodeById((int) Configuration::get('PS_CURRENCY_DEFAULT'));
+            $orderId = null;
+            if ($request->query->has('order_id')) {
+                $orderId = (int) $request->query->get('order_id');
+            }
```

Comment:
```suggestion
            $orderId = (int) $request->query->get('order_id', null);
```
Use default value instead

Reply:
I thought about it but it wouldn't work because null would be cast into int `0`

Reply:
Ohhh you're right :sweat_smile: